scripts/setup-lcs.sh

```diff
@@ -53,6 +53,14 @@ apply_resources() {
     else
         kubectl create configmap lightspeed-stack --from-file="$ROOTDIR"/tmp/lightspeed-stack.yaml -n "$DEPLOYMENT_NAMESPACE"
     fi
+    if kubectl get configmap llama-stack-config -n "$DEPLOYMENT_NAMESPACE" >/dev/null 2>&1; then
```

Comment:
sometimes i wonder if we should re-create CM if resources are present in resources/